e2e/mobile/utils/runCli.ts

```diff
@@ -20,10 +22,14 @@ function extractFlagValue<K extends keyof LiveDataOpts>(
   return idx !== -1 && idx + 1 < parts.length ? parts[idx + 1] : undefined;
 }
 
-export function runCliCommand(command: string): Promise<string> {
+export function runCliCommand(command: string, speculosAddress?: string): Promise<string> {
```

Comment:
Aren't we already waiting for speculos in previous steps ?
Why do we need to add speculos address again in env ?